src/core-interfaces.ts

```diff
@@ -207,6 +207,7 @@ export interface IImageOpts extends PositionOptions, OptsDataOrPath {
 	hyperlink?: HyperLink
 	rounding?: boolean
 	placeholder?: any
+	rotate?: number
```

Comment:
Does this also need to be added to `types/index.d.ts` @gitbrent?

Reply:
Yes, good catch!